controller/user-controller.js

```diff
@@ -0,0 +1,41 @@
+const {userService} = require('../services/user-services');
+
+const signUp = async (req, res) => {
+  try {
+    const {email, password, firstname, lastname, skintype} =req.body;
+
+    await userService.signUp(email, password, firstname, lastname, skintype);
+
+    if (!email|| !password || !firstname || !lastname){
+      throw new Error("KEY Error!");
+    }
+
+    return res.status(201).send({message: "SIGNUP_SUCCESS"});
+  }catch (err) {
```

Comment:
마이너한 수정 (공백을 통해 가독성을 높여주세요.)
```suggestion
  } catch (err) {
```